docs/docs/how-to/start-developing.md

```diff
@@ -0,0 +1,167 @@
+
+!!! note
+    You are free to use any code editor of your choice, although you will have to complete steps of this tutorial through alternate means.
+
+## Setting up the GCC Compiler ##
+
+A compiler translates human-readable code into machine code. We will be setting up the GCC compiler so that our code written in the human-readable C++ language can be understood by the computers!
+
+First, [download MSYS2](https://www.msys2.org/) and install it with the default settings. MSYS2 helps to set up MinGW-w64, which uses the GCC compiler to create Windows programs.
+
+When the MinGW terminal pops up, enter `pacman -Sy --needed --noconfirm base-devel mingw-w64-x86_64-toolchain mingw-w64-x86_64-cmake mingw-w64-x86_64-ninja mingw-w64-x86_64-clang mingw-w64-x86_64-lld wget`
+
+This will install the Mingw-w64 toolset which includes GCC, and other build tools that will be employed later.
+
+```
+pacman -Sy --needed --noconfirm base-devel mingw-w64-x86_64-toolchain mingw-w64-x86_64-cmake mingw-w64-x86_64-ninja mingw-w64-x86_64-clang mingw-w64-x86_64-lld wget
+```
+
+
+Next, we will set the Mingw-w64 path to the environment variables. This will allow the terminal to search the Mingw-w64 directory for executables to run without requiring the user to specify the full address of the .exe each time.
+
+Search for “Edit environment variables for your account.” in the Window’s search bar and select the option that appears.
+
+!!! help
+    You can also access the Environment Variables window from "Control Panel → User Accounts → User Accounts → Change my environment variables".
+
+Double-click on “Path” under “User variables”.
+
+![Double-click on the “Path” user variable.](assets/set-environment-variable-mingw64-path.png)
+
+In the “Edit environment variable” window, select “New” and add `C:\msys64\mingw64\bin` to the empty field that appears.
+```
+C:\msys64\mingw64\bin
+```
+
+![Add the mingw64 bin directory to the environment variables.](assets/set-environment-variable-mingw64-new.png)
+
+## Rust ##
+
+Rust is a programming language that we will be using. Install it [here](https://www.rust-lang.org/tools/install). 
+
+Open up the Command Prompt. You can use the Windows Search Bar to find the Command Prompt application.
+
+![Search for the Command Prompt in the Window’s search bar.](assets/search-command-prompt.png)
+
+!!! tip
+    You can check if Rust is properly installed by inputting `cargo –version` in the Command Prompt. 
+
+    If a version number is printed, your installation was successful!
+        ```
+        cargo –version
+        ```
+
+Input `rustup default nightly` to use the nightly version of Rust. The nightly release is updated more frequently compared to the stable and beta versions.
+	```
+    rustup default nightly
+	```
+
+Input the command `rustup target add wasm32-unknown-unknown --toolchain nightly` to add support for building to web browsers.
+	```
+    rustup target add wasm32-unknown-unknown --toolchain nightly
```

Comment:
It's correct but rust users are more familiar with this "kind-of shortcut" version:
```suggestion
    rustup +nightly target add wasm32-unknown-unknown
```